app/models/user.rb

```diff
@@ -0,0 +1,24 @@
+# == Schema Information
+#
+# Table name: users
```

Comment:
We don't need new table users, we already have 2 existing types of users: Teachers and Students
please set up pundit around these 2 users and modify models, migrations, and other settings accordingly